packages/gallery/src/components/helpers/slideshowVirtualization.ts

```diff
@@ -0,0 +1,56 @@
+import { GALLERY_CONSTS, Options } from 'pro-gallery-lib';
+
+export function getItemsInViewportOrMargin({
+  groups,
+  options,
+  galleryWidth,
+  activeIndex,
+}: {
+  groups: any[];
+  options: Options;
+  galleryWidth: number;
+  activeIndex: number;
+}) {
+  const {
+    enabled = false,
+    rightItemMargin = 3,
+    leftItemMargin = 3,
+    rightItemScrollMargin = 10,
+    leftItemScrollMargin = 10,
+  } = options.behaviourParams?.gallery?.horizontal?.itemVirtualization || {};
```

Comment:
@yhattav is it necessary to check all the `?` stuff? Aren't we assuring new SP will always be present?